meg_runtime/ui/addpluginpanel.py

```diff
@@ -0,0 +1,130 @@
+from PyQt5 import QtWidgets
+
+from meg_runtime.ui.manager import UIManager
+from meg_runtime.plugins import PluginManager
+from meg_runtime.ui.basepanel import BasePanel
+
+class AddPluginPanel(BasePanel):
+    """Setup the plugin panel."""
+
+    selected_file = None

```

Comment:
I very much doubt that you want this to be a class variable and not an instance variable. You should be setting this on self instance in __init__(), then using self.selected_file.